client/app/scripts/reducers/root.js

```diff
@@ -721,6 +724,34 @@ export function rootReducer(state = initialState, action) {
       return state.set('showingTroubleshootingMenu', !state.get('showingTroubleshootingMenu'));
     }
 
+    case ActionTypes.TOGGLE_CONTRAST_MODE: {
+      const modules = [
+        require.resolve('../../styles/main.scss'),
+        require.resolve('../../styles/contrast.scss')
+      ];
+      // Bust the webpack require cache to for a re-download of the stylesheets
+      modules.forEach((i) => {
+        const children = require.cache[i] ? require.cache[i].children : [];
+        children.forEach((c) => {
+          delete require.cache[c];
+        });
+        delete require.cache[i];
+      });
+
+      const contrastEnabled = state.get('contrastMode');
+      if (!contrastEnabled || action.enabled) {
```

Comment:
That makes sense but I think you still missed a part of my comment :) Let me put it this way:

`contrastEnabled && action.enabled` -> `contrast.scss`
`!contrastEnabled && !action.enabled` -> `contrast.scss`
`!contrastEnabled && action.enabled` -> `contrast.scss`
`contrastEnabled && !action.enabled` -> `main.scss`

3/4 scenarious would result in `contrast` styles being chosen over `main` which feels wrong. I tried ignoring the `contrastEnabled` variable and this seemed to work just fine:

```
if (action.enabled) {
  require.ensure([], () => {
    require('../../styles/contrast.scss');
  });
} else {
  require.ensure([], () => {
    require('../../styles/main.scss');
  });
}
```

Could you perhaps also verify that on your side? Not sure I'm missing some special circumstances like hot-reloading, etc...